src/gui/search/searchsortmodel.h

```diff
@@ -49,6 +49,11 @@ class SearchSortModel final : public QSortFilterProxyModel
         NB_SEARCH_COLUMNS
     };
 
+    enum SearchDataRole
+    {
+        UnderlyingDataRole = Qt::UserRole
```

Comment:
It should be declared in underlying model since its really define such a role.

Reply:
here too Standard Model is used,